CONTRIBUTING.md

```diff
@@ -23,21 +23,22 @@ guidelines](https://help.github.com/en/github/site-policy/github-community-guide
 
 ## Development process
 
-Development happens against the `master` branch following the [GitHub flow
-model](https://guides.github.com/introduction/flow/). Contributors create
-feature branches off of `master`, and their pull requests should target the
-`master` branch. Maintainers are responsible for prompt review of pull
-requests.
+Development happens against the `master` branch following the
+[GitHub flow model](https://guides.github.com/introduction/flow/).
+Contributors create feature branches off of `master`, and their pull requests
+should target the `master` branch. Maintainers are responsible for prompt
+review of pull requests.
 
```

Comment:
We should maybe mention the fact that you do it on your own fork, right?

Reply:
You don't really have a choice if you're not part of the existing "team" (as in the GitHub term) due to permissions, but I'm happy to add that for clarity.